src/components/UI/Icons/Correct.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
Does this belong in this PR? Adding correct/wrong applies to all types of questions, no?